src/inmanta/moduletool.py

```diff
@@ -507,6 +508,14 @@ def modules_parser_config(cls, parser: ArgumentParser) -> None:
             default=None,
             dest="output_dir",
         )
+        build.add_argument(
+            "--dev",
+            dest="dev_build",
+            help="Perform a development build of the module. This adds the build tag `.dev<timestamp>` to the "
+            "package name. The timestamp has the form %%Y%%m%%d%%H%%M%%S.",
+            default=False,
+            action="store_true",
+        )
```

Comment:
* This `--dev` options is indeed not used at the moment. The idea is to use it in https://github.com/inmanta/irt/issues/1186 for modules for which we don't create a release tag on every commit. Next to that it will be useful for end users to have the possibility to create dev releases.
* The parsing of the arguments happens here https://github.com/inmanta/inmanta-core/blob/01f957fa5be0df0ba6d67d4094837c5e0f9b82fe/src/inmanta/app.py#L765